docs/Contributors_Guide/continuous_integration.rst

```diff
@@ -496,15 +496,33 @@ development testing. If testing is done on a stable release, then the
 corresponding MET stable release will be used. However, there may be an
 instance where a change in MET breaks something in another METplus component,
 i.e. METplotpy or METviewer, until a corresponding change is made to that
-component. If this occurs then some of the METplus use cases may break. To
-allow the tests to run successfully in the meantime, an option was added to
-force the version of the MET tag that is used to build the METplus Docker image
-that is used for testing. In the **testing.yml** workflow file,
-there is a commented variable called
-MET_FORCE_TAG that can be uncommented and set to force the version of MET to
-use. This variable is found in the **get_image** job under the **env** section
+component. If this occurs then some of the METplus use cases may break.
+
+Another situation that may require a different MET Docker image is if there
+are changes in a MET feature or bugfix branch that are needed to test changes
+in METplus.
+
+To allow the tests to run successfully in these cases, an option was added to
+force a specific MET Docker image to be used to build the METplus Docker image
+that is used for testing.
+
+In the **testing.yml** workflow file, there is a commented variable called
+SET_MET_IMAGE that can be uncommented and set the MET Docker image to use.
+This variable is found in the **get_image** job under the **env** section
 for the step named "Get METplus Image."
 
+The format of the value is <REPO>:<TAG>
+where the DockerHub repo used is dtcenter/<REPO> and the tag used is <TAG>.
+
+Stable releases of MET are found in the dtcenter/met DockerHub repo and
```

Comment:
Where is the "dtcenter/met DockerHub repo"?

Reply:
On DockerHub here: https://hub.docker.com/repository/docker/dtcenter/met/general

Reply:
Added link